Convert to markdown: distributed.tensor.parallel.rst, distributed.tensor.rst, distributions.rst, dlpack.rst

ggsmith842 opened the pull request:
Fixes #155019 

## Description
Convert to markdown: distributed.tensor.parallel.rst, distributed.tensor.rst, distributions.rst, dlpack.rst

## Checklist
- [X] dlpack.rst converted to dlpack.md --> [Preview](https://docs-preview.pytorch.org/pytorch/pytorch/155297/dlpack.html)
- [X] distributions.rst converted to distributions.md --> [Preview](https://docs-preview.pytorch.org/pytorch/pytorch/155297/distributions.html) 
- [X] distributed.tensor.rst converted to distributed.tensor.md --> [Preview](https://docs-preview.pytorch.org/pytorch/pytorch/155297/distributed.tensor.html)
- [X] distributed.tensor.parallel.rst converted to distributed.tensor.parallel.md --> [Preview](https://docs-preview.pytorch.org/pytorch/pytorch/155297/distributed.tensor.parallel.html)

cc @svekars @sekyondaMeta @AlannaBurke

ggsmith842 committed:
convert distributed rst to md

ggsmith842 commented:
@pytorchbot label "topic: not user facing"

ggsmith842 commented:
@pytorchbot label "topic: docs"

svekars committed:
Update docs/source/distributed.md

ggsmith842 committed:
Merge branch 'pytorch:main' into main

svekars committed:
Update docs/source/distributed.md

svekars commented on docs/source/distributed.tensor.md:
````diff
@@ -0,0 +1,250 @@
+
+```
+
+(create-dtensor)=
````
```suggestion
(create_dtensor)=
```

svekars committed:
Update docs/source/distributed.tensor.md

svekars reviewed (commented):
This looks pretty good - a few minor issues I commented on and it looks like distributed.tensor.md and distributed.tensor.parallel were not renamed correctly as their creation dates have changed - can you please fix those.

ggsmith842 committed:
Apply suggestions from code review

Co-authored-by: Svetlana Karslioglu <svekars@meta.com>

ggsmith842 committed:
update distributions.md

ggsmith842 commented on docs/source/distributions.md:
````diff
@@ -1,460 +1,776 @@
+
+% for tracking purposes
+
+```{eval-rst}
````
Fixed

ggsmith842 committed:
revert distributed.tensor.parallel to main version and then convert to md

ggsmith842 committed:
restore original files with create dates from main

ggsmith842 committed:
run git mv to rename rst to md without any changes

ggsmith842 committed:
fix create date to align with main

ggsmith842 committed:
convert to md from rst

ggsmith842 commented on docs/source/distributed.tensor.md:
````diff
@@ -0,0 +1,250 @@
+```{eval-rst}
````
I think its resolved now. Had a ton of trouble with `git mv` on this file and the other one.

svekars commented on docs/source/distributions.md:
````diff
@@ -1,460 +1,690 @@
 
+```{eval-rst}
 .. currentmodule:: torch.distributions.mixture_same_family
+```
+
+```{eval-rst}
````
this should fix build [error](https://github.com/pytorch/pytorch/actions/runs/15549382341/job/43834218730?pr=155297#step:11:1075).
```suggestion

```

ggsmith842 committed:
Update distributions.md

Co-authored-by: Svetlana Karslioglu <svekars@meta.com>

ggsmith842 committed:
fixed issues in distributions.md causing build fail

ggsmith842 committed:
Update distributions.md

Co-authored-by: Svetlana Karslioglu <svekars@meta.com>

svekars commented on docs/source/distributed.tensor.parallel.md:
````diff
@@ -0,0 +1,96 @@
+```{eval-rst}
````
I've seen this issue and this is what you might try like this:
Save your .md changes somewhere. Then from docs/source dir:
```
git checkout HEAD -- distributed.tensor.parallel.rst
git mv distributed.tensor.parallel.rst distributed.tensor.parallel.md
git commit -m "Renamed distributed.tensor.parallel.rst to distributed.tensor.parallel.md"
```
You can check the log after that: `git log --follow -- distributed.tensor.parallel.md`
It should show you a bunch of commits. You can send an update to this PR with this.
Then copy and paste your .md changes/save/comit/push.

ggsmith842 replied:
@svekars so create two new ones and close this one?

ggsmith842 committed:
add missing  ..py:module

ggsmith842 committed:
remove distributed.md and revert to original

ggsmith842 committed:
remove distributed.md and revert to original

ggsmith842 committed:
revert distributed.tensor and distributed.tensor.parallel

ggsmith842 committed:
run git mv on dist.tensor and dist.tensor.parallel

ggsmith842 committed:
Update distributed.tensor.md

Added actual markdown content; trying to prevent it saying it deleted and created a new file instead of showing the rename

ggsmith842 committed:
Update distributed.tensor.md

Added actual markdown content; trying to prevent it saying it deleted and created a new file instead of showing the rename

ggsmith842 committed:
Merge branch 'main' of https://github.com/ggsmith842/pytorch

ggsmith842 committed:
Revert "run git mv on dist.tensor and dist.tensor.parallel"

This reverts commit 0620701a7ca025dbaf99d67285a559f248663222.

ggsmith842 committed:
run git mv on dist.tensor and dist.tensor.parallel

ggsmith842 committed:
rename distributed.rst to md with git mv

ggsmith842 committed:
add minor change to test mv vs add/delete

ggsmith842 committed:
add another minor change

ggsmith842 committed:
add another minor change

ggsmith842 committed:
add another minor change

ggsmith842 committed:
add another minor change

ggsmith842 committed:
Merge branch 'main' of https://github.com/ggsmith842/pytorch

ggsmith842 committed:
Revert "add another minor change"

This reverts commit bf5962e1d16512166e1a0f0be42eee37cef7a411.

ggsmith842 committed:
Revert "add minor change to test mv vs add/delete"

This reverts commit 04bf72deb8c2e634a0df15d57af99f18b8c20b58.

ggsmith842 committed:
minor change to hidded section

ggsmith842 committed:
changes  to title

ggsmith842 committed:
update distributed.tensor.md

ggsmith842 committed:
update distributed.tensor.md

ggsmith842 committed:
Merge branch 'main' of https://github.com/ggsmith842/pytorch

ggsmith842 committed:
Revert "changes  to title"

This reverts commit 63a231ab362d9d69b446e9d4bdd92f335fbefe42.

ggsmith842 committed:
revert distributed.tensor

ggsmith842 committed:
rename distributed.tensor.rst to md

ggsmith842 committed:
minor changes

ggsmith842 committed:
minor changes

ggsmith842 committed:
minor changes

ggsmith842 committed:
fix bottom part of parallel

ggsmith842 committed:
Update distributed.tensor.md

ggsmith842 committed:
update distributed.tensor.md

ggsmith842 committed:
update distributed.tensor.parallel.md

svekars reviewed (approved):
LGTM! Thanks for all the fixes!

svekars commented:
@pytorchbot merge -f "Docs change only. Doc signal is green"
